.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+dist: trusty
+language: node_js
+notifications:
+  email: false
+  slack: false
+node_js:
+  - "8"
+after_success:
+  npm run semantic-release
```

Comment:
lol, noooo.. you've gone the other way!

Reply:
Let's use `npx` here, and drop it from our deps

Reply:
🤣